src/benchmark/operators/table_scan_sorted_benchmark.cpp

```diff
@@ -0,0 +1,269 @@
+#include <memory>
+
+#include "../micro_benchmark_basic_fixture.hpp"
+#include "benchmark/benchmark.h"
+#include "expression/expression_functional.hpp"
+#include "operators/table_scan.hpp"
+#include "operators/table_wrapper.hpp"
+#include "storage/segment_encoding_utils.hpp"
+#include "storage/table.hpp"
+#include "table_generator.hpp"
+#include "table_scan_sorted_benchmark.hpp"
+#include "type_cast.hpp"
+#include "utils/load_table.hpp"
+
+using namespace opossum::expression_functional;  // NOLINT
+
+namespace opossum {
+
+const int string_size = 512;
+
+void _clear_cache() {
+  std::vector<int> clear = std::vector<int>();
+  clear.resize(500 * 1000 * 1000, 42);
+  for (uint i = 0; i < clear.size(); i++) {
+    clear[i] += 1;
+  }
+  clear.resize(0);
+}
+
+std::shared_ptr<TableColumnDefinitions> create_column_definitions(const DataType data_type) {
```

Comment:
Keep in mind that you're in the opossum namespace and that generically named functions like `create_column_definitions` might well appear in other translation units and might conflict. Consider anonymous namespaces